frontend/src/features/admin-form/responses/IndividualResponsePage/PaymentSection.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Divider, Link, Stack, Text } from '@chakra-ui/react'
+import { keyBy } from 'lodash'
+
+import { SubmissionPaymentDto } from '~shared/types'
+
+import { getPaymentDataView } from '../common/utils/getPaymentDataView'
+
+const PaymentDataItem = ({
+  name,
+  value,
+  isMonospace,
+  isUrl,
+}: {
+  name: string
+  value: string
+  isMonospace?: boolean
+  isUrl?: boolean
+}) => (
+  <Stack direction={{ base: 'column', md: 'row' }}>
+    <Text textStyle="subhead-1">{name}:</Text>
+    <Text
+      sx={
+        isMonospace
+          ? {
+              fontFeatureSettings: "'tnum' on, 'lnum' on, 'zero' on, 'cv05' on",
+            }
+          : {}
+      }
+    >
+      {isUrl ? (
+        <Link as="a" href={value} target="_blank">
```

Comment:
`as="a"` is the default if, probably don't have to set this, unless it's overwritten elsewhere

Reply:
thanks! removed in [5722388](https://github.com/opengovsg/FormSG/pull/6061/commits/5722388cf319829eccbb107346de6434f9365694)